### ISSUE TITLE
na_ontap_vscan_scanner_pool  only gets one scanner pool

### ISSUE THREAD

**Issue by markrob** [ansible-collections/netapp#19]
The current na_ontap_vscan_scanner_pol module (added in 2.8) only gets a single child scanner pool as opposed to all children scanner pools.

The code snippet below shows the issue:

if result.get_child_by_name('num-records') and int(result.get_child_content('num-records')) >= 1:
            if result.get_child_by_name('attributes-list').get_child_by_name('vscan-scanner-pool-info').get_child_content(
                    'scanner-pool') == self.scanner_pool:
                return result.get_child_by_name('attributes-list').get_child_by_name('vscan-scanner-pool-info')

The code snippet below shows the modification we have made to the new module to get all scanner pools i.e. get_children():

if result.get_child_by_name('num-records') and int(result.get_child_content('num-records')) >= 1:
            children = result.get_child_by_name('attributes-list').get_children()
            for child in children:
                if child.get_child_content('scanner-pool') == self.scanner_pool:
                    return child
            return False

**Comment by lonico** [comment]
I'm not sure I understand the issue.
The module was using
```
scanner_pool_info.add_new_child('scanner-pool', self.scanner_pool)
```
in the query, so this should have limited to scope of the scanner pool of interest.
(which makes the check 
```
if result.get_child_by_name('attributes-list').get_child_by_name('vscan-scanner-pool-info').get_child_content(
'scanner-pool') == self.scanner_pool:
```
redundant.

Do you still see this issue with recent collections?

**Comment by lonico** [comment]
fix in 19.11.0

na_ontap_vscan_scanner_pool: fix module only gets one scanner pool.
